charts/karmada/values.yaml

```diff
@@ -148,7 +153,7 @@ scheduler:
   image:
     registry: docker.io
     repository: karmada/karmada-scheduler
-    tag: latest
+    tag: *karmadaImageVersion
```

Comment:
We always use {{xxx}} as a placeholder. Can we use it here?

Reply:
no, this is the `values.yaml`, helm can not perform secondary rendering.

Reply:
I mean placeholder for sed replacement

Reply:
do you mean:
```console
# in value.yaml
tag: {{ karmadaImageVersion }}

# in package-helm-chart.sh
sed -i'' -e "s/{{ karmadaImageVersion }}/{{ ${version} }}/g" ./charts/"${chart}"/values.yaml
```

I have also considered this approach, but I am not very recommending it.

Because there are many `tag: {{ karmadaImageVersion }}`, and may also be misused in other places in the future, apply `sed` to all is too violent. Besides, one may mistakenly think that `{{ karmadaImageVersion }}` is also done through rendering at first glance, actually it is not by rendering, it shall better be distinguished.

Reply:
Beside, use `tag: {{ karmadaImageVersion }}`, how can you handle default value `latest` ?

Reply:
Make sense.

> how can you handle default value latest.

With sed.

Reply:
"with sed" can not resolve default value in local installation, for local installtion is just one command based on source code:
```
helm install karmada -n karmada-system --create-namespace --dependency-update ./charts/karmada
```